### PR TITLE
DI-1242 | AllowUserRegistration configuration updated.

### PR DESCRIPTION
**Pull request by lchavarriaga**

### CODE REVIEW

File: DataImport.Web/Infrastructure/MinimumRequiredSetupValidationFilter.cs [hunk 1]
```diff
@@ -31,25 +27,14 @@ public MinimumRequiredSetupValidationFilter(DataImportDbContext dbContext)
...
-            {
-                filterContext.Result = new RedirectToRouteResult(
-                    new RouteValueDictionary
-                    {
-                            { "controller", "Configuration" },
-                            { "action", "Index" },
-                            { nameof(EditConfiguration.Query.OdsApiServerException), true }
-                    });
-            }
         }
```
Comment (CSR2017) [inline DataImport.Web/Infrastructure/MinimumRequiredSetupValidationFilter.cs:38]: @lchavarriaga 
Please consider to verify the flow - if no api server connections defined, then the page needs to be redirected to API-connections page.

File: DataImport.Web/Infrastructure/MvcTransactionFilter.cs [hunk 2]
```diff
@@ -65,16 +64,6 @@ private static void HandleOdsApiServerException(ActionExecutedContext filterCont
...
-            else
-            {
-                result = new RedirectToRouteResult(
-                    new RouteValueDictionary
-                    {
-                        { "controller", "Configuration" },
-                        { "action", "Index" },
-                        { nameof(EditConfiguration.Query.OdsApiServerException), true }
-                    });
-            }
```
Comment (CSR2017) [inline DataImport.Web/Infrastructure/MvcTransactionFilter.cs:77]: @lchavarriaga
Please consider to verify the flow - if no api server connections defined, then the page needs to be redirected to API-connections page.

Comment (lchavarriaga) [reply DataImport.Web/Infrastructure/MvcTransactionFilter.cs:77]: Fixed

Comment (lchavarriaga) [reply DataImport.Web/Infrastructure/MvcTransactionFilter.cs:77]: Fixed

File: DataImport.Web/Infrastructure/MinimumRequiredSetupValidationFilter.cs (hunk 1 above)
Comment (lchavarriaga) [reply DataImport.Web/Infrastructure/MinimumRequiredSetupValidationFilter.cs:49]: Fixed

**Review by CSR2017 — APPROVED** [review]

**Pull request merged**